benchmarks/conftest.py

```diff
@@ -0,0 +1,152 @@
+from attr import dataclass
+import json
+import pytest
+
+from ariadne import QueryType, gql, make_executable_schema
+
+
+@dataclass
+class Person:
+    name: str
+    age: int
+
+
+@dataclass
+class User:
+    id: int
+    name: str
+    group: dict
```

Comment:
Group should be other (nested) Dataclass

Reply:
Now group and avatar are other Dataclasses